builtin/providers/aws/resource_aws_efs_file_system.go

```diff
@@ -149,15 +149,34 @@ func resourceAwsEfsFileSystemRead(d *schema.ResourceData, meta interface{}) erro
 		return fmt.Errorf("EFS file system %q could not be found.", d.Id())
 	}
 
-	tagsResp, err := conn.DescribeTags(&efs.DescribeTagsInput{
-		FileSystemId: aws.String(d.Id()),
-	})
-	if err != nil {
-		return fmt.Errorf("Error retrieving EC2 tags for EFS file system (%q): %s",
-			d.Id(), err.Error())
+	tags := make([]*efs.Tag, 0)
+	var marker string
+	for {
+		params := &efs.DescribeTagsInput{
+			FileSystemId: aws.String(d.Id()),
+		}
+		if marker != "" {
+			params.Marker = aws.String(marker)
+		}
+
+		tagsResp, err := conn.DescribeTags(params)
+		if err != nil {
+			return fmt.Errorf("Error retrieving EC2 tags for EFS file system (%q): %s",
+				d.Id(), err.Error())
```

Comment:
A thought: log the failure but `break` instead of returning an error. This could allow Terraform to continue creating all the things, but possibly show a diff later because there was some error in retrieving all the tags. 

Just a thought, maybe you feel failing to retrieve tags here is correct, so I trust/defer to your call here